setup/ConfigEditor.xaml.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Microsoft.Win32;
+using Microsoft.WindowsAPICodePack.Dialogs;
+using System.ComponentModel;
+using System.IO;
+using System.Windows;
+
+namespace ReShade.Setup
+{
+	/// <summary>
+	/// Interaction logic for ConfigEditor.xaml
+	/// </summary>
+	public partial class ConfigEditor : Window
+	{
+		private const string INI_SECTION = "GENERAL";
+
+		private string _configFilePath;
+
+		public ConfigEditor(string configPath)
+		{
+			InitializeComponent();
+			_configFilePath = configPath;
+
+			_load();
+			Closing += (object sender, CancelEventArgs args) => _save();
+		}
+
+		private void _writeValue(string key, string value)
```

Comment:
I'll go back through and fix the styling. It's been a while since I've worked with .NET and just went with my general preferences since there wasn't a style guide.

It'd be nice if somebody could add a basic style guide to the README, or somewhere, documenting at least indentation, naming conventions, and braces placement.